portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,112 @@
     <script src="script.js"></script>
   </head>
+
   <body>
+    
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+     <p class="me">
+         <h1 style="color:#5e7059;font-family:optima;font-size:300%;">Carine Gordillo
+            <img src="/images/plant.png" width="250" height="300" style="float:right"></h1>
+
+         <div id="header">
+             <style>
+             img{border-radius: 30%;}
+             </style>
+             <p id="me"><img src="/images/Mainme.jpg" alt="Avatar" style="vertical-align:middle;margin:0px 50px;width:200px"></p>
+         </div>
+         <h3 style="color:#5e7059;font-size:130%;"></h3style>About Me!</h3>
+         <p style="color:#000000;">Hello! I'm Carine (KAh-dín)  and I live in Downey, California <br>but parents are originally from Zapopan, Jalisco. I am 19 <br>years old and attend 
+            Cal State Long Beach. Currently, I am <br> participating in a progam by Google called SPS and I am on team 41! </p>
+
+      <hr>
+      <h3 style="color:#5e7059;font-size:130%;">Hobbies!</h3>
+      <p class="hobbies">
+          <b style="color:#5e7059;line-height:30px;">Sports</b>
+          <br style="color:#5e7059;line-height:60px;"> I really enjoy playing any sport because of how competative I am. I did <br> play Soccer for 9 
+          years and Volleyball for my highschool senior season.</br>
+         <b style="color:#5e7059;line-height:30px">Baking<img src="/decor/bluebutterflies.png" width="100" height="125" style="float:right"></b>
+         <br style="color:#5e7059;line-height:60px;">For sure something I am trying to get better at but I love being able to bake <br>and I really want to start getting into making food in general:)</br>
+         <b style="color:#5e7059;line-height:30px">DIY</b>
+         <br style="color:#5e7059;">I love doing projects whether its something I need or something jsut for<br> fun, I find it really enjoyable</br>
+      <hr>
+      <h3 style="color:#5e7059;font-size:130%;">Music!</h3>
+      <p class= "music">
+        
+          <b style="color:#5e7059;line-height:30px">Lauryn Hill&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;N.W.A&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Kid Cudi
+          <br style="color:#5e7059;line-height:30px;">J.Cole&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+          &nbsp;&nbsp;&nbsp;Camila 
```

Comment:
I'm assuming this line would look something line "J.Cole----------Camila" on your website, where "-" is a space? Another (cleaner) way to do this, would be to have two `div`s, once for each artist, and style them so they're next to each other. This will let you get rid of the nbsp

Flexboxes (https://css-tricks.com/snippets/css/a-guide-to-flexbox/) are probably the best way to do this in industry, but whatever way you can get working to style these horizontally (without the nbsp) works fine with me. If you're doing the css refactor in a followup pull request, you can do this change in the followup pull request as well.